src/librawspeed/common/table17.inc

```diff
@@ -0,0 +1,285 @@
+// Taken from https://github.com/gopro/gpr/raw/a513701afce7b03173213a2f67dfd9dd28fa1868/source/lib/vc5_common/table17.inc
+/*!	@file table17.inc
+
+	Codebook automatically generated by Huffman program
+	Created: Wed May 11 2005 21:23:08 UTC
+
+	Modified by hand to simplify for use with the reference decoder.
+	
+	Note that the codebook entries with a run length of zero are
+	special codewords that are used to mark significant locations
+	in the encoded bitstream.
+
+	(c) 2013 Society of Motion Picture & Television Engineers LLC and Woodman Labs, Inc.
```

Comment:
I don't think this will fly

Reply:
IANAL, but as the first comment line states, this file was taken as-is from the gopro/gpr repository, which is dual-licensed under Apache2 or MIT so it should be compatible. As I don't have access to the original specs, there is not much else I can do about this.

Reply:
Fixed now:
https://github.com/gopro/cineform-sdk/blob/master/Common/table17.inc

Reply:
Sorry, wrong link:
https://github.com/gopro/gpr/blob/d2d78d1b236a278a13ed5abee6870af2b490c89f/source/lib/vc5_common/table17.inc

Reply:
Thanks, i suppose that is better / as good as it will get.
I guess i now need to read up about using "dual-licensed under Apache2 or MIT" in LGPL library.